packages/gamut-labs/src/landingPage/PageHeroMedia.tsx

```diff
@@ -0,0 +1,44 @@
+import { Column, Video } from '@codecademy/gamut';
+import { breakpoints } from '@codecademy/gamut-styles';
+import styled from '@emotion/styled';
+import { omit } from 'lodash';
+import React from 'react';
+
+import { MediaProps } from './PageHero';
+
+const RightColumn = styled(Column)`
+  margin-left: 1rem;
+
+  @media only screen and (max-width: ${breakpoints.sm}) {
+    display: none;
+  }
+`;
+
+const VideoColumn = styled(Column)`
+  margin-left: 1rem;
+`;
+
+const Image = styled.img`
+  width: 100%;
+`;
+
+const StyledVideo = styled(Video)`
+  align-self: center;
+`;
+
+export const PageHeroMedia: React.FC<MediaProps> = (media) => {
+  if (media.type === 'image') {
+    return (
+      <RightColumn size={3}>
+        <Image src={media.src} alt={media.alt} />
+      </RightColumn>
+    );
+  } else if (media && media.type === 'video') {
+    const videoArgs = omit(media, 'type');
+    return (
+      <VideoColumn size={{ xs: 12, sm: 7 }}>
+        <StyledVideo {...videoArgs} />
+      </VideoColumn>
+    );
+  } else return null;
```

Comment:
This case can never be hit because the media always is either image or video, right?

```suggestion
  }
```

Reply:
![Screen Shot 2021-01-27 at 11 37 10 AM](https://user-images.githubusercontent.com/22219812/106022648-12b1ca80-6094-11eb-9390-8be507dd70ff.png)

I was getting this typescript error, returning `null` fixed. But maybe there is a better way to fix it?

Reply:
Heh, annoying. Looks like TypeScript doesn't have this particular situation memorized as one where it can ignore that last case... If you change to a `switch (media.type) {` it does. 🤷

Reply:
I `switch`ed the if statement for a `switch` instead